### PR TITLE
[BACKLOG-17671] Parquet Dialogs layout

### PR DESCRIPTION
**Pull request by tgf**

### CODE REVIEW

File: kettle-plugins/formats/src/main/resources/org/pentaho/big/data/kettle/plugins/formats/parquet/messages/messages_en_US.properties [hunk 1]
```diff
@@ -0,0 +1,9 @@
+BaseStepDialog.StepName=Step name:
```
Comment (ddiroma) [inline kettle-plugins/formats/src/main/resources/org/pentaho/big/data/kettle/plugins/formats/parquet/messages/messages_en_US.properties]: @tgf I believe colons should be escaped in property files??

Comment (tgf) [inline kettle-plugins/formats/src/main/resources/org/pentaho/big/data/kettle/plugins/formats/parquet/messages/messages_en_US.properties]: @ddiroma don't think so, at least they're showing ok

Comment (ddiroma) [inline kettle-plugins/formats/src/main/resources/org/pentaho/big/data/kettle/plugins/formats/parquet/messages/messages_en_US.properties]: @tgf Sorry. Should have been more specific. I think Ben had some issues with them showing on Linux. What OS do you use?

Comment (tgf) [inline kettle-plugins/formats/src/main/resources/org/pentaho/big/data/kettle/plugins/formats/parquet/messages/messages_en_US.properties]: I'm on linux. Do you remember where that was? We have a lot of unescaped colons in kettle

Comment (ddiroma) [inline kettle-plugins/formats/src/main/resources/org/pentaho/big/data/kettle/plugins/formats/parquet/messages/messages_en_US.properties]: I do not remember exactly where it was. I will merge and then I am sure QA will test on all OSs. Maybe he had issues where it showed on Linux, but not on windows or mac. I'll merge either way. Thx for the comments :)

**Pull request merged**